Sage-Bionetworks/Bridge-Exporter
# More Mutable Schemas

Now that Synapse table columns are mutable, we should propagate that change to Bridge schemas. See https://sagebionetworks.jira.com/browse/BRIDGE-1863

This change allows certain field type changes and allows an increase in string lengths.

Testing done:
* mvn verify (unit tests, Jacoco test coverage, findbugs)
* integ tests
* manual tests

Manual test cases:
* modify a schema to add fields and re-order other fields
* modify a schema to modify fields and re-order other fields
* change a field from bool to int
* change a field from int to float
* change a field from int to timestamp
* change a field from float to string
* increase string length
